curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-049.md

```diff
@@ -0,0 +1,210 @@
+---
+id: 60b69a66b6ddb80858c515a6
+title: Part 49
+challengeType: 0
+dashedName: part-49
+---
+
+# --description--
+
+FontAwesome icons come with their own styling to define the icon. However, you can still set the styling yourself as well, to change things like the color and size. For now, use a `class` selector to target your `fa-music` icons. Set the `display` to `inline-block`, the `margin-top` to `8%`, and the `margin-left` to `13%`.
```

Comment:
Why `inline-block`? Default `block` seems to work just fine.

Reply:
🤷🏻 That's what was in the prototype.